lib/bot.js

```diff
@@ -120,6 +127,18 @@ const TwitchBot = class TwitchBot extends EventEmitter {
     this.irc.write(text + "\r\n")
   }
 
+
+
+  join (channel){
+    channel = parser.formatCHANNEL(channel);
+    this.writeIrcMessage(`JOIN ${channel}`);
```

Comment:
should we append the joined channel to `this.channels`?

Reply:
Yes, any new channels joined (after we're sure it's connected to the IRC channel) should be appended to `this.channels`